pkg/library/kubernetes/provision.go

```diff
@@ -34,11 +35,16 @@ import (
 // Only Kubernetes/OpenShift components that are inlined are supported; components that define
 // a URI will cause a FailError to be returned
```

Comment:
Documentatioon should be changed to `...a URI will cause a WarningError to be returned`